fract4d/test_codegen.py

```diff
@@ -1768,7 +1772,7 @@ def mycotanh(z):
         #print tests
         
         # CONSIDER: Python 2.7 thinks this cotanh(0) is -nan,-nan, older versions think (nan,nan)
-        tests[0][2] = "(-nan,-nan)"
```

Comment:
Since we know that some environments produce -nan and some produce nan, is it possible to adjust the test logic so that we would accept either? 
 think in general the distinction between -nan and nan is pretty academic, so if we could make it so either was accepted throughout, that would be ideal.

Reply:
That nan/-nan value comes from C++, and that's why I changed the C++ code produced by  `inspect_complex` function to make sure we always get "nan".
This functions `cotanhtests` and `cotantests` are called by the test `test_stdlib` which at the end relies on the mentioned ìnspect_complex` to generate the C++ that is compiled and executed to check the return values.